### ISSUE TITLE
aoc: Deleting broken link for shared folder

### ISSUE THREAD

**Issue by knapsu** [IBM/aspera-cli#116]
Aspera CLI version: `3.x`, `4.x`

Problem:
I am unable to delete an orphaned link to shared folder. We can get such broken/invalid link if the user who shared a folder with us (adds as a collaborator) has revoked the permissions or has deleted the source folder. 

There is a command `ascli aoc files delete ...` but if we try to delete a broken link then we will receive an error "404 Not Found".
I would expect that a broken link would get removed.

**Comment by laurent-martin** [comment]
Did you try to delete it using "admin" rights, i.e. using the access key secret ?

```bash
ascli aoc admin res node do name '_name_of_node_here' --secret='_node_secret_here' br /
```

(one can also use the numerical node id: `node do 12345 ...`

For that you need to find out the node on which the broken link.
and then navigate to the shared folder link location on that node.

**Comment by knapsu** [comment]
Hi Martin. Thank you for the reply.

Before creating a issue on GitHub I've reached out to IBM Support and gave instructions how to reproduce the issues I have with Shared Folders and Aspera CLI client  for Aspera On Cloud. What I got is a WebEx call, where a sales person tried to convince me that there is nothing wrong with how Aspera works despite admitting that I can end up with broken links and unmanageable folder hierarchy... Unproductive call. But I was also told that I could try making an enhancement request on IBM portal or feature request directly on GitHub CLI repo.

I've chosen to report the problems here directly as IBM support was not good and maybe other people that can only interact with AOC using the CLI client will end up in similar problems I got hit with. I was able to workaround the issues by getting access to Aspera on Cloud web interface where I could remove the broken links. CLI seems to be missing this, which I would expect to be a basic functionality.

Thank you for fixing the typo in the issue title.
